move.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, _pid: int, _type: str, _value: int | str = ""):
 
     def __str__(self):
         return (
-            f"[{self.pid}: {self.type} ({self.value})]"
+            f"[{self.pid}: {self.type} (dupa{self.value})]"
```

Comment:
The word _dupa_ is listed among [polish profanities](https://en.wikipedia.org/wiki/Polish_profanity). Please consider changing it to something more appropriate.

Reply:
Thank you @Ponewor for making this site a more inclusive place. You'll never know what problematic words are there in all the world languages.